front/src/config/i18n/fr.json

```diff
@@ -1983,7 +1983,7 @@
   },
   "httpErrors": {
     "rateLimitError": "Vous avez essayé ce formulaire trop de fois. Veuillez réessayer dans 30 minutes ou redémarrer votre instance Gladys maintenant.",
-    "unknownError": "Une erreur inconnue s'est produite. Veuillez réessayer ou contacter Gladys community.",
+    "unknownError": "Une erreur inconnue s'est produite. Veuillez réessayer ou nous contacter sur la <a href=\"https://community.gladysassistant.com\">communauté Gladys</a>.",
```

Comment:
It'll not work if the component is not modified.

Also, don't forget  target=\"_blank\" rel=\"noopener noreferrer\"